core/src/main/resources/error/error-classes.json

```diff
@@ -1062,6 +1062,11 @@
       "Failed to set original permission <permission> back to the created path: <path>. Exception: <message>"
     ]
   },
+  "ROOT_CONVERTER_RETURNS_NULL" : {
```

Comment:
This error class is pretty specific to JSON parser. Let's add the `JSON` word to the name. For example, `INVALID_JSON_ROOT_FIELD`